objects.js

```diff
@@ -141,6 +141,15 @@ var StagePrompterMorph;
 var Note;
 var SpriteHighlightMorph;
 
+function defaultFiniteValue(num, def) {
```

Comment:
okay, can we move these functions from the global namespace into the constructors they work in? Here it would be SpriteMorph, with a shadow in StageMorph?